From d25ab4c89e6ce3afdb84db0c9028c7f5facbbf3c Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sun, 16 Sep 2018 16:38:48 +0000
Subject: [PATCH 21/25] Ignore broken and non-applicable tests

long-linker-command-lines: takes >10 minutes to run (but still passes)
sparc-struct-abi: no sparc target
sysroot-crates-are-unstable: can't run rustc without rpath
---
 src/test/codegen/sparc-struct-abi.rs                            | 1 +
 src/test/run-make-fulldeps/long-linker-command-lines/Makefile   | 1 -
 src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile | 2 +-
 3 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/test/codegen/sparc-struct-abi.rs b/src/test/codegen/sparc-struct-abi.rs
index 56c4364d59..9aebf8f002 100644
--- a/src/test/codegen/sparc-struct-abi.rs
+++ b/src/test/codegen/sparc-struct-abi.rs
@@ -13,6 +13,7 @@
 
 // only-sparc64
 // compile-flags: -O --target=sparc64-unknown-linux-gnu --crate-type=rlib
+// ignore-test
 #![feature(no_core, lang_items)]
 #![no_core]
 
diff --git a/src/test/run-make-fulldeps/long-linker-command-lines/Makefile b/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
index 5876fbc94b..e9f5c33b77 100644
--- a/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
+++ b/src/test/run-make-fulldeps/long-linker-command-lines/Makefile
@@ -2,4 +2,3 @@
 
 all:
 	$(RUSTC) foo.rs -g -O
-	RUSTC="$(RUSTC_ORIGINAL)" $(call RUN,foo)
diff --git a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
index 9e77070685..7784230b46 100644
--- a/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
+++ b/src/test/run-make-fulldeps/sysroot-crates-are-unstable/Makefile
@@ -1,4 +1,4 @@
 -include ../tools.mk
 
 all:
-	env '$(HOST_RPATH_ENV)' python2.7 test.py
+	true
-- 
2.19.1

